spec/index/index_spec.rb

```diff
@@ -280,4 +280,11 @@
       it { expect { idx.at 2, 3 }.to raise_error IndexError }
     end
   end
-end
\ No newline at end of file
+
+  context "Enumerable" do
```

Comment:
Can you follow the convention set in the rest of the file? The top level context is the method name in this case, followed by tests for that method that test particular functionality.